src/resources/TagType.js

```diff
@@ -46,17 +46,13 @@ export default class TagType extends CrudSetBase {
     return 'tag-types';
   }
 
-  get _Child () {
-    return Tag;
-  }
-
   /**
-   * Get items associated with the set
+   * Get the list of tags that are attached to this type
    * @returns {SimpleResourceProxy} - A proxy for accessing the resource
    */
-  get items () {
+  get tags () {
     const data = { [this.constructor.foreignKeyName]: this.id };
```

Comment:
Does this static variable exist in `CrudBase`?

Reply:
Interestingly it does not but somehow this still works, it is located in `CrudSetBase`